### ISSUE TITLE
Face Reenactment

### ISSUE THREAD

**Issue by RAJA-PARIKSHAT** [or-toledano/animation-with-keypoint-mask#2]
Can I use this code to train on face videos data, basically I want to create a real-time application where I can generate smiling and blink videos in real-time based on driving videos? @Dov-Gertz @or-toledano

**Comment by or-toledano** [comment]
> Can I use this code to train on face videos data, basically I want to create a real-time application where I can generate smiling and blink videos in real-time based on driving videos? @Dov-Gertz @or-toledano


Yes. You would need to train on a face dataset instead of a full body dataset, but you can just change Tai Chi to VoxCeleb and it would work because the method isn't domain specific.
You can copy a current config and change the root_dir to your local VoxCeleb dataset
If there are any problems take 
https://github.com/AliaksandrSiarohin/first-order-model/blob/master/config/vox-256.yaml
for reference too
